Sources/NIOWebSocket/NIOWebSocketClientUpgrader.swift

```diff
@@ -92,3 +97,25 @@ public final class NIOWebSocketClientUpgrader: NIOHTTPClientProtocolUpgrader {
         }
     }
 }
+
+extension NIOWebSocketClientUpgrader {
+    /// Generates a random WebSocket Request Key by generating 16 bytes randomly and encoding them as a base64 string as defined in RFC6455 https://tools.ietf.org/html/rfc6455#section-4.1
+    /// - Parameter generator: the `RandomNumberGenerator` used as a the source of randomness
+    /// - Returns: base64 encoded request key
+    public static func randomRequestKey<Generator>(
```

Comment:
Can we make this `@inlinable`? That will help with the perf cost of the generic.

Reply:
Done ✅